src/pages/trade-types/multiplier/_start-trading.tsx

```diff
@@ -26,27 +26,21 @@ const StartTrading = () => {
         <PatternContainer background="white" padding="4rem 0">
             <SmallContainer direction="column" ai="flex-start">
                 <Header as="h3" size="3.2rem" mb="4rem">
-                    {localize('Start trading multipliers on Deriv')}
+                    <Localize translate_text="_t_Start trading multipliers on Deriv_t_" />
                 </Header>
                 <FixTimeline>
                     <FixTimeline.Item title={<Localize translate_text="Practise" />}>
```

Comment:
Can we refactor it as well? Instead of passing `Localize` as title component lets pass title and in timeline pass title in `Localize` component
By this approach we prevents misunderstanding and cod complexity 😄

Reply:
title is a only for attribute in item component, so instead i will pass it with `localize()` function